docs/developers/msgs/staging/subspaces/edit-subspace.md

```diff
@@ -35,8 +39,10 @@ The `subspace_type` field will only accept the following values:
   "id": "19de02e105c68a60e45c289bff19fde745bca9c63c38f2095b59e8e8090ae1af",
   "owner": "desmos1tqzrfy9ujrk883e2wezsumyvq64gcm65vhdyr7",
   "name": "mooncake",
+  "description": "a good cake with secret messages in it",
+  "logo": "https://mooncake-logo-png.com",
+  "subspace_type": "<Indicates if users can post in it freely or not>",
```

Comment:
Can you please provide an example of valid types? I think it's also better to list them